tensorflow/python/keras/losses.py

```diff
@@ -1843,15 +1843,35 @@ def deserialize(name, custom_objects=None):
 
 @keras_export('keras.losses.get')
 def get(identifier):
-  """Retrieves a Keras loss function.
+  """Retrieves a Keras loss as a `function`/`Loss` class instance.
+
+  You can get loss as a `function` using parameter `identifier` as a string
+  of the loss function as shown in below example.
+  >>> loss = tf.keras.losses.get("categorical_crossentropy")
+  >>> type(loss)
+  function
+
+  You can get loss as a `Loss` class instance using parameter `identifier`
+  as a string of the class name of the loss.
+  >>> loss = tf.keras.losses.get("CategoricalCrossentropy")
+  >>> type(loss)
+  tensorflow.python.keras.losses.CategoricalCrossentropy
+
+  You can also specify `config` of the loss to this function by passing dict
+  containing `class_name` and `config` as an identifier.
```

Comment:
Also note that the class name must map to a `Loss` class

Reply:
Done